.github/workflows/generate-measurements.yml

```diff
@@ -20,6 +20,12 @@ on:
         type: boolean
         required: true
 
+env:
+  ARM_CLIENT_ID: "b657a00e-813a-4dc7-9b09-fa498a254d71"
+  ARM_CLIENT_SECRET: ${{ secrets.AZURE_E2E_CLIENT_SECRET }}
+  ARM_SUBSCRIPTION_ID: "0d202bbb-4fa7-4af8-8125-58c269a05435"
+  ARM_TENANT_ID: "adb650a8-5da3-4b15-b4b0-3daf65ff7626"
```

Comment:
Can be removed now, since the values are injected via secrets

Reply:
Oops, how did I forget this one... Fixed, thanks!